### PR TITLE
add actions and reducers

### LINKED ISSUE #10: Create reducer for details

### PR DESCRIPTION
**Pull request by Quinn-H**
close #10

### CODE REVIEW

File: src/actions/details.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const showDetails = (companyId) => {
+  return {
+    payload: companyId,
+    type: 'SHOW_DETIALS'
```
Comment (davidchristie) [inline src/actions/details.js]: Please change this to:
`type: 'SHOW_DETAILS'`

File: src/actions/details.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  }
+}
+
+export const hideDetials = (companyId) => {
```
Comment (davidchristie) [inline src/actions/details.js]: Please change to `hideDetails`

File: src/actions/details.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return {
+    payload: companyId,
+    type: 'HIDE_DETIALS'
```
Comment (davidchristie) [inline src/actions/details.js]: And here as well

File: src/reducers/details.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const inintalState = {
```
Comment (davidchristie) [inline src/reducers/details.js]: Please name this `initialState`

File: src/reducers/details.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  companyId: null
+}
+
+export default (state = inintalState, {payload, type}) => {
+  switch (type) {
+    case 'SHOW_DETAILS':
+      return {
+        companyId: payload
+      }
+    case 'HIDE_DETIALS':
```
Comment (davidchristie) [inline src/reducers/details.js]: Should be `'HIDE_DETAILS'`

File: src/containers/Board/Board.js [hunk 6]
```diff
@@ -23,7 +22,6 @@ class Board extends Component {
             return <Column column={column} key={index} />
           })}
         </Grid.Row>
-        <Details />
```
Comment (davidchristie) [inline src/containers/Board/Board.js]: No need to remove this component, please add it back.

**Review by davidchristie — CHANGES_REQUESTED** [review]

File: src/reducers/details.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const inintalState = {
+  companyId: null
+}
+
+export default (state = inintalState, {payload, type}) => {
```
Comment (amandahogan) [inline src/reducers/details.js]: please fix name `state = initialState`

File: src/containers/Board/Board.js [hunk 8]
```diff
@@ -3,8 +3,10 @@ import { connect } from 'react-redux'
 import { Button, Grid, Icon, Menu } from 'semantic-ui-react'
 
 import { addColumn, removeColumn } from '../../actions/columns'
-import Column from '../Column'
 import Details from '../../components/Details'
+
```
Comment (davidchristie) [inline src/containers/Board/Board.js]: Please delete this empty line (7)

**Review by davidchristie — CHANGES_REQUESTED** [review]

**Review by amandahogan — APPROVED** [review]

**Review by davidchristie — APPROVED** [review]

**Pull request merged**